integration/jnifuse/fs/src/main/java/alluxio/jnifuse/AbstractFuseFileSystem.java

```diff
@@ -59,9 +59,10 @@ public AbstractFuseFileSystem(Path mountPoint) {
    *
    * @param blocking whether this command is blocking
    * @param debug whether to show debug information
+   * @param maxIdleThreads maximum number of idle threads enforced by libfuse3
    * @param fuseOpts
    */
-  public void mount(boolean blocking, boolean debug, String[] fuseOpts) {
+  public void mount(boolean blocking, boolean debug, int maxIdleThreads, String[] fuseOpts) {
```

Comment:
I would expect there is a MountConfiguration class holding all those information, and the whole class object can be formed when launches Fuse and all intermediate steps just directly pass this object or take information from it (without modifying)